src/cf-expandables.less

```diff
@@ -686,7 +686,7 @@
     border-bottom: 1px solid @expandable-group-divider;
     background: @expandable-group_header-bg;
     color: @expandable-group_header-text;
-    .h5();
+    .heading-5();
     margin-bottom: 0;
```

Comment:
It is, `.heading-5` has [a margin bottom](https://github.com/cfpb/cf-core/blob/gh-pages/src/cf-base.less#L183)